app/assets/stylesheets/docs.scss

```diff
@@ -0,0 +1,117 @@
+$blue: #53ADC6;
+$red: #CF726E;
+$green: #5BC89E;
+$white: #f0f0f0;
+
+html,
+body {
+  font-family: 'Lato', serif;
+  font-size: 18px;
+  line-height: 1.62em;
+  margin: 0;
+}
+
+img {
+  max-width: 100%;
+}
+
+h1,
+h2,
+h3 {
+  font-family: 'Fjalla One', sans-serif;
+  margin-top: 2em;
+}
+
+code {
+  font-size: .8rem;
+  padding: .4em .2em 0;
+  font-family: 'Source Code Pro';
+}
+
+a {
+  color: $blue;
+  text-decoration: none;
+}
+
+.content {
+  margin-bottom: 2em;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 2em;
+  width: 61em;
+
+  .sidebar {
+    float: left;
+    width: 20em;
+
+    &-links {
+      list-style: none;
+      padding-left: 0;
+    }
+  }
+
+  .main {
+    float: right;
+    width: 40em;
+  }
+}
+
+.flash-wrapper {
+  background-color: $red;
+  color: $white;
+  padding: 1em;
+
+  .flash {
+    margin-left: auto;
+    margin-right: auto;
+    width: 40em;
+  }
+}
+
+table {
+  font-size: 1rem;
+  width: 100%;
+
+  thead {
+    text-align: left;
+  }
+
+  th {
+    border-bottom: 1px solid #aaa;
+  }
+
+  th, td {
+    padding: .5em;
+    padding-left: 0.5em;
+    padding-right: 2.5em;
+    &:last-child { padding-right: 0.5em; }
+  }
+
+  tr:nth-child(2n) {
+    background-color: #eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.